source/ed/connectors/gtfs_parser.cpp

```diff
@@ -861,7 +861,12 @@ void TripsGtfsHandler::handle_line(Data& data, const csv_row& row, bool) {
 
     nm::Line* line = it->second;
 
-    nm::Route* route = get_or_create_route(data, {line, row[direction_id_c]});
+    // direction_id is optional (and possible values "0" or "1"), so defaulting to "0"
+    std::string direction_id = "0";
```

Comment:
empty string is not gtfs complient, no?
this was a quick fix, but if both work...